### PR TITLE
fix: remove unused tech_account_email

### PR DESCRIPTION
**Pull request by shazron**
See https://github.com/adobe/aio-lib-ims/issues/63

## How Has This Been Tested?

npm test

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Review by purplecabbage — APPROVED** [review]
lgtm!

File: src/ims-jwt.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function configMissingKeys (configData) {
   }
 
   const missingKeys = []
-  const requiredKeys = ['client_id', 'client_secret', 'technical_account_email', 'technical_account_id', 'meta_scopes', 'ims_org_id', 'private_key']
+  const requiredKeys = ['client_id', 'client_secret', 'technical_account_id', 'meta_scopes', 'ims_org_id', 'private_key']
```
Comment (moritzraho) [inline src/ims-jwt.js:28]: will the plugin support a configuration that still has `technical_account_email` in the context? 
Thinking about backward compatibility + being able to generate a token from the .aio jwt config loaded by the app plugin (which still includes the email)

Comment (shazron) [reply src/ims-jwt.js:28]: it should, since `technical_account_email` being in the context will just be something extra and not used.

Comment (moritzraho) [reply src/ims-jwt.js:28]: ok perfect 👍 thx

**Review by moritzraho — APPROVED** [review]
lgtm

**Pull request merged**